tracing/opencensus.go

```diff
@@ -140,6 +140,9 @@ func WithExporter(name string, logger *zap.SugaredLogger) ConfigOption {
 //   "host" is a endpoint IP like activator-service's endpoint IP.
 func WithExporterFull(name, host string, logger *zap.SugaredLogger) ConfigOption {
 	return func(cfg *config.Config) error {
+		if cfg == nil {
+			return errors.New("nil config")
+		}
```

Comment:
Prevent the `Finish()` function from throwing `panic`. More info in https://github.com/knative/eventing/pull/6219#discussion_r837277395

Reply:
Ehm. This will not work. This function needs to reach [this line](https://github.com/knative/pkg/blob/main/tracing/opencensus.go#L182) when calling Finish even if the passed config is nil.

Reply:
Pushed another commit that fixes that.